exporter/opentelemetry-exporter-jaeger-thrift/tests/test_jaeger_exporter_thrift.py

```diff
@@ -62,9 +62,15 @@ def setUp(self):
             is_remote=False,
         )
 
-        self._test_span = trace._Span("test_span", context=self.context)
-        self._test_span.start()
-        self._test_span.end()
+        self._test_span = trace._Span(
+            "test_span",
+            context=self.context,
+            # Use a fixed version because a longer/shorter version number
+            # might break tests that care about packet size.
+            resource=Resource.create({"telemetry.sdk.version": "0.0.0.dev0"}),
+        )
+        self._test_span.start(start_time=1)
+        self._test_span.end(end_time=3)
```

Comment:
I'm not sure if thrift is like protobuf where variable length integers are used, but I think it's good to eliminate non-determinstic elements here, so I use a fixed start/end time.